spec/services/flow/steps/linked_applications/copy_step_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe Flow::Steps::LinkedApplications::CopyStep do
+  let(:legal_aid_application) { create(:legal_aid_application, copy_case:) }
+  let(:copy_case) { true }
+
+  describe "#path" do
+    subject { described_class.path.call(legal_aid_application) }
+
+    it { is_expected.to eql "/providers/applications/#{legal_aid_application.id}/link_application/copy?locale=en" }
```

Comment:
If we agree to proceed with url helpers (_path) rather than explicit urls, maybe this could be changed?

Reply:
Good spot - I wrote this before we had the conversation! 😄